addon/components/frost-list.js

```diff
@@ -134,6 +136,20 @@ export default Component.extend({
     }
   },
 
+  didReceiveAttrs () {
```

Comment:
I would put this in the `init`, not the receive, since there should be no expectation of your `itemKey` changing